config/delta/rules.js

```diff
@@ -109,7 +109,7 @@ export default [
     match: {
       predicate: {
         type: 'uri',
-        value: 'http://mu.semte.ch/vocabularies/ext/bevatDocumentversie' // Will get changed in future model refactoring
```

Comment:
Deze prov#generated, we gebruiken die op veel plaatsen. Dat gaat vermoedelijk resulteren in veel meer onnodige delta's die worden verstuurd. Kan dat kwaad ?

Reply:
Zoals eerder besproken: je analyse klopt, maar het kan geen kwaad. :ok: